tools/noise/gen-profile

```diff
@@ -504,7 +504,7 @@ if [ "$pdf_enabled" ]; then
 		done
 	done
 	echo Filelist: $flist
-	pdftk $flist cat output $resultpdf
+	pdfcat $resultpdf $flist
```

Comment:
Exactly, even if the output should be identical for simple concatenation, I'd rather not change the tool for those who are used to have pdftk